### ISSUE TITLE
Miner closes stream when rejecting deals

### ISSUE THREAD

**Issue by whyrusleeping** [filecoin-project/lotus#5686]
Currently, when a miner rejects a storage deal, they just close the stream and dont send a polite response to the client. We should always send some sort of a repsonse to the client when rejecting their deals

**Comment by PatrickDeuse** [comment]
a user defines string would be preferred

**Comment by whyrusleeping** [comment]
Yeah, that would be nice. If the deal decider program could return a message that the miner sends to the user.

**Comment by PatrickDeuse** [comment]
external filters can. but being able to define a default message would be nice to

**Comment by TippyFlitsUK** [comment]
Hi 👋 

The Legacy Lotus Markets sub-system [reached EOL](https://github.com/filecoin-project/lotus/releases/tag/v1.18.0) at the end of the 31st January 2023.

This ticket is being marked as `won't fix` and closed as the Lotus Team will no longer be making any further fixes or enhancements to the legacy markets subsystem.

Please feel free to re-open this ticket in the new Boost markets sub-system repository at https://github.com/filecoin-project/boost if you feel that it is still relevant.

Many thanks 🙏
